### PR TITLE
Add random suffix to GKE cluster name to avoid conflict.

### PR DESCRIPTION
**Pull request by TeweiLuo**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
/kind cleanup

> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Make GKE cluster names more or less unique to avoid conflicts where the same boskos project was used.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/k8s-integration/main.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ var (
 	numNodes           = flag.Int("num-nodes", -1, "the number of nodes in the test cluster")
 	imageType          = flag.String("image-type", "cos", "the image type to use for the cluster")
 	gkeReleaseChannel  = flag.String("gke-release-channel", "", "GKE release channel to be used for cluster deploy. One of 'rapid', 'stable' or 'regular'")
-	gkeTestClusterName = flag.String("gke-cluster-name", "gcp-pd-csi-cluster", "GKE cluster name")
+	gkeTestClusterName = flag.String("gke-cluster-name", "pdcsi", "GKE cluster name")
```
Comment (verult) [inline test/k8s-integration/main.go:50]: Change flag comment to something like "Prefix to the GKE cluster name. A randomly generated 4-character suffix is appended."

Comment (TeweiLuo) [reply test/k8s-integration/main.go]: Done.

**Pull request merged**